src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py

```diff
@@ -5,3 +5,11 @@
 
 RESOURCE_PROVIDER = 'Microsoft.ContainerRegistry'
 RESOURCE_TYPE = RESOURCE_PROVIDER + '/registries'
+
+ALLOWED_ROLES = [
+    'owner',
+    'contributor',
+    'reader'
+]
+
+DEFAULT_ROLE = 'Reader'
```

Comment:
They are case insensitive. The array is normalized role names for comparison. DEFAULT_ROLE will be shown to the user and 'Reader' looks nicer than 'reader', I think.